src/pages/basic_data/app_admin/CustomerInstitutionInformationFromCristin.tsx

```diff
@@ -0,0 +1,94 @@
+import { Grid, TextField } from '@mui/material';
+import { useQuery } from '@tanstack/react-query';
+import { useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { fetchOrganization } from '../../../api/cristinApi';
+import { PageSpinner } from '../../../components/PageSpinner';
+import { Organization } from '../../../types/organization.types';
+import { getUnitTopLevelCode } from '../../../utils/institutions-helpers';
+import { getLanguageString } from '../../../utils/translation-helpers';
+
+interface CustomerInstitutionInformationFromCristinProps {
+  cristinId: string | undefined;
+  customerData: Organization | null;
+  setName: (string: string) => void;
+  displayName: string | undefined;
```

Comment:
Bare fyi at dette er det samme. Mer lettlest IMO, men det er jo subjektivt :)
```suggestion
  cristinId?: string;
  customerData: Organization | null;
  setName: (string: string) => void;
  displayName?: string;
```